c7n/resources/awslambda.py

```diff
@@ -257,6 +258,28 @@ def process(self, resources, event=None):
             resources, event)
 
 
+@AWSLambda.filter_registry.register('kms-key')
+class KmsFilter(KmsRelatedFilter):
+    """
+    Filter a resource by its associcated kms key and optionally the aliasname
+    of the kms key by using 'c7n:AliasName'
+
+    :example:
+
+        .. code-block:: yaml
+
+            policies:
+                - name: lambda-kms-key-filters
+                  resource: lambda
```

Comment:
minor nit, generally we want examples to lead folks in the right direction.. qualify aws prefixes with aws.lambda.. because the form is intended to be provider.resource

Reply:
Ok thanks! Sure thing, I'll remember that in the future. Updated in a5cec18.